sfcasts/crud-controller.md

```diff
@@ -1,171 +1,90 @@
+
+Oh! but before we do, we actually want to go to "Override Methods..." and select `configureActions`. Now we want to call the parent method that can create the actions object and set up all of those default actions for us. To do that, add an action to our index page. We can use that CRUD class call on page index.
+
+```terminal
+->add(Crud::PAGE_INDEX)
+```
+
+ We're in, and we can actually use a constant for this as well.
+ 
+ ```terminal
+Action::DETAIL
+ ```
+
+So on the index page, I want to add the action and then the `DETAIL` action. The result of this, when we refresh the `QuestionCreditController`, is that we have this "Show" link that goes to the `DETAIL` action, and we can go to any section. 
+
+And just like that, we've modified every single credit controller in the system! Easy peasy! *But*, since this entity is so simple, let's disable the `DETAIL` action for just this topic section. To do that, open up `TopicCrudController.php`, and just like before, we'll go to "Code Generate" or "cmd + N" on a Mac and go to "Override Methods..." and override that same `configureActions`.
+
+By the time this method is called, it will pass us. The actions of that was already set up by our dashboard, so it will already have the `DETAIL` action enabled for the index page. But *now*, we can control that by saying:
+
+```terminal
+->disable(Action::DETAIL)
+```
+
+We're gonna talk more about the actions configuration, but these are the main things that you can do inside of them: You can add a new action to a page, *or* you can just completely disable an action globally for all of your pages. As soon as we refresh here, our `DETAIL` action is gone, but if we go to our other sections, it's *still* there.
+
+The big takeaway here is that everything is going through our `DashboardController`, which means that we can configure things on a dashboard level, which will apply to all of our CRUD, *or* we can configure things for one specific CRUD. The fact that all of the CRUD controllers go through this `/avenue`, where all has one other effect related to security, means that all of our controllers are already secure. That's thanks to our `access_control`.
+
+Remember back in `config/packages/security.yaml`, we had a little access control here that said if the URL starts with `/admin`, require `roll_admin`. So all of these automatically, without doing anything else, require `role_admin`. We'll talk more later about how to secure different admin controllers with different roles, but at the very least, you have to have `role_admin` to get anywhere, which is awesome.
+
+But one important point: Adding this `access_control` was necessary. Why? Even though the URL is `/admin`, when you go to a `crudController` like this, it does not execute our `DashboardController` index action. This can be a little confusing.
```

Comment:
The "this" in "Adding this `access_control`..." sounds kind of out of place. Might read better without it. Happened to stick out to me while re-reading everything.

Reply:
And maybe "does not" should be "doesn't"?

Reply:
The "this" in this case IS needed, but it's complex. `access_control` is a list of things, and I'm talking about how we added one specific one. So `this` refers to the ONE `access_control` that was necessary :)

> And maybe "does not" should be "doesn't"?

Yup, I usually like contractions better! Easier to say!